smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/CommandGenerator.java

```diff
@@ -153,10 +153,12 @@ private void generateCommandMiddlewareResolver(String configType) {
             // Resolve the middleware stack.
             writer.write("\nconst stack = clientStack.concat(this.middlewareStack);\n");
             writer.write("const { logger } = configuration;");
+            writer.write("const clientName = '$L';", symbolProvider.toSymbol(service).getName());
```

Comment:
I see they're pulled from the code online 158/159, but these `'$L'` can be replaced with an unquoted `$S` to generate quoted strings.

Reply:
Resolved in [`4285d9e` (#230)](https://github.com/awslabs/smithy-typescript/pull/230/commits/4285d9eb2076d2a3462c0d6af46ae2e0c8ec7bdd)